courses/free-courses-en.md

```diff
@@ -335,6 +335,7 @@
 * [A Tour Of Go](https://tour.golang.org/welcome/1)
 * [Go Tutorial](http://www.tutorialspoint.com/go/index.htm)
 * [Golang \| Gin HTTP Framework](https://www.youtube.com/playlist?list=PL3eAkoh7fypr8zrkiygiY1e9osoqjoV9w) - Pragmatic Reviews
+* [Golang in under an hour (2021)](https://www.youtube.com/watch?v=N0fIANJkwic) - Eli Goldberg
```

Comment:
Not needs year/version tagging since it isn't valuable and ancient

```suggestion
* [Golang in under an hour](https://www.youtube.com/watch?v=N0fIANJkwic) - Eli Goldberg
```